e2e-tests/utils/walletPageHelper.ts

```diff
@@ -37,6 +37,132 @@ export default class WalletPageHelper {
       .click()
   }
 
+  /**
+   * Onboard using walletPageHelper
+   */
+  async onboardWithSeedPhrase(recoveryPhrase: string): Promise<void> {
+    const onboardingPage = await getOnboardingPage(this.context)
+    await this.onboarding.addAccountFromSeed({
+      phrase: recoveryPhrase,
+      onboardingPage,
+    })
+    await this.setViewportSize()
+    await this.goToStartPage()
+  }
+
+  async verifyTopWrap(network: RegExp, accountLabel: RegExp): Promise<void> {
+    // TODO: maybe we could also verify graphical elements (network icon, profile picture, etc)?
+
+    await expect(
+      this.popup.getByTestId("top_menu_network_switcher").last()
+    ).toHaveText(network)
+    await this.popup
+      .getByTestId("top_menu_network_switcher")
+      .last()
+      .click({ trial: true })
+
+    await this.popup.locator(".connection_button").last().click({ trial: true })
+
+    await expect(
+      this.popup.getByTestId("top_menu_profile_button").last()
+    ).toHaveText(accountLabel, { timeout: 240000 })
+    await this.popup
+      .getByTestId("top_menu_profile_button")
+      .last()
+      .click({ trial: true })
+    // TODO: verify 'Copy address'
+  }
+
+  async verifyBottomWrap(): Promise<void> {
+    await this.popup
+      .getByLabel("Main")
+      .getByText("Wallet", { exact: true })
+      .click({ trial: true })
+    await this.popup
+      .getByLabel("Main")
+      .getByText("NFTs", { exact: true })
+      .click({ trial: true })
+    await this.popup
+      .getByLabel("Main")
+      .getByText("Portfolio", { exact: true })
+      .click({ trial: true })
+    await this.popup
+      .getByLabel("Main")
+      .getByText("Settings", { exact: true })
+      .click({ trial: true })
+  }
+
+  /**
+   *  The function checks elements of the main page that should always be present.
+   */
+  async verifyCommonElements(
+    network: RegExp,
+    testnet: boolean,
+    accountLabel: RegExp
+  ): Promise<void> {
```

Comment:
Non-blocking: this is opaque enough at the call site that I'd like to suggest we move to an object instead:

```suggestion
  async verifyCommonElements({
    network, testnet, accountLabel
  }: {
    network: RegExp,
    testnet: boolean,
    accountLabel: RegExp
  ): Promise<void> {
```

This would make the invocations much clearer IMO:

```typescript
  walletPageHelper.verifyCommonElements({ network: /.../, testnet: true, accountLabel: /.../ })
```